changelogs/unreleased/2023-02-02-release-bump-compiler-version.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: Bump COMPILER_VERSION to 2023.1
+change-type: patch
+destination-branches: [master]
```

Comment:
For some reason I can't attach a suggestion... We'll need to include iso6 because they are still aligned. Which also means that ideally we want to have OSS and iso6 both in the RC stage before releasing the stable for either. It's not terribly important, but it prevents the creation of second (very small) version for the second of the two.